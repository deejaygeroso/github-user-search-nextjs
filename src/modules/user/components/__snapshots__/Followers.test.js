import React from 'react';
import Followers from '../Followers';
import renderer from "react-test-renderer";

it("renders correctly", () => {
  const followerList = {
      allIds: [], byId: {}
  }
  const tree = renderer
    .create(<Followers followerList={followerList} />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
