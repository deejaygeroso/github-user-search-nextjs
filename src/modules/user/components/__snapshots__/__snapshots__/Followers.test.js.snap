// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<ul
  className="MuiList-root-3 MuiList-padding-4 Followers-root-1"
/>
`;
